keyboards/mwstudio/cover60/config.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2023 TW59420 <https://github.com/TW59420>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define FORCE_NKRO
+
+#define WS2812_DI_PIN B8
+#define RGBLED_NUM 19
```

Comment:
These can be moved to info.json.

Reply:
I moved to info.json,But the compilation does not pass. [QMK Firmware](https://docs.qmk.fm/#/)not updated

Reply:
`FORCE_NKRO` -> `usb.force_nkro`
`WS2812_DI_PIN` -> `ws2812.pin`
`RGBLED_NUM` -> `rgblight.led_count`